content/altinn-studio/guides/monitor-and-instrument/_index.en.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Monitor and instrument
+linktitle: Monitor and instrument
+description: Get started with Altinn monitoring tools and instrumentation
+toc: true
+weight: 990
+---
+
+{{% notice info %}}
+This page refers to configuration when [OpenTelemetry (OTel)](https://opentelemetry.io/) is enabled in the app from v8 and newer.
```

Comment:
I think this message is important but a bit akwardly formulated.
Two points need to come across here 
- version >= 8 is required for OTel.
- For app insights guides, don't look here. 

 If there is existing documentation on app insights one could consider a child-page structure where one section has the title OTel, and the other has a title that mentions application insights. This would leave only the information about v8 in the info box.